web/ui/mantine-ui/src/pages/AlertsPage.tsx

```diff
@@ -460,15 +455,13 @@ export default function AlertsPage() {
           </Alert>
         )
       )}
-      <Stack>
```

Comment:
Just a quick drive-by cleanup here, getting rid of a superfluous `<Stack>`, that's the only real change in this file 😬